Sources/Clappr/Classes/Plugin/Playback/AVFoundationPlayback.swift

```diff
@@ -227,6 +227,10 @@ open class AVFoundationPlayback: Playback {
     private var canStartAt: Bool {
         return startAt != 0.0 && playbackType == .vod
     }
+    
+    private var canLiveStartTime: Bool {
+        return isEpochInsideDVRWindow(liveStartTime) && isDvrAvailable
```

Comment:
We could short circuit this if we invert the clause's order. Avoiding doing the math if DVR isn't available.

Reply:
Done